From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aaron Friel <mayreply@aaronfriel.com>
Date: Thu, 22 Jun 2023 09:45:36 -0700
Subject: [PATCH 07/10] Exclude scripts dir from provider go module

This directory references a private Go module, which we cannot "go vet" or check.

diff --git a/scripts/go.mod b/scripts/go.mod
index 3aea06471..ddd2cb8ff 100644
--- a/scripts/go.mod
+++ b/scripts/go.mod
@@ -1,3 +1,3 @@
-module github.com/hashicorp/terraform-provider-google-beta/scripts
+module exclude
 
 go 1.20
